llpc/context/llpcCompiler.cpp

```diff
@@ -728,6 +728,246 @@ template <typename T> static FormattedBytes formatBytesLittleEndian(ArrayRef<T>
   return format_bytes(bytes, /* FirstByteOffset = */ None, /* NumPerLine = */ 16, /* ByteGroupSize = */ 1);
 }
 
+// Build shader from the specified info.
+//
+// @param [in]  pipelineInfo : Info to build this shader module
+// @param [out] pipelineOut : Output of building this shader module
+// @param [in]  stage : Shader stage of needing to compile
+// @param pPipelineDumpFile : Handle of pipeline dump file
+//
+// @returns : Result::Success if successful. Other return codes indicate failure.
```

Comment:
nit: please fix the documentation: missing separator line (`====`), no need to specify `[in]` only, extra line before `@returns`. Also in other documentation comments.

Reply:
I think it is better to add another change to fix it.

Reply:
> I think it is better to add another change to fix it.

For the existing code in this file, fully agreed. For new code, I'd strongly prefer to get it right from the beginning.